docs/OV_Converter_UG/prepare_model/convert_model/Convert_Model_From_TensorFlow.md

```diff
@@ -0,0 +1,311 @@
+
+   .. tab-item:: Python
+      :sync: py
+
+         import openvino as ov
+         ov_model = ov.convert_model('your_model_file.pb')
+
+   .. tab-item:: CLI
+      :sync: cli
+
+         ovc your_model_file.pb
+
+
+Converting Non-Frozen Model Formats
++++++++++++++++++++++++++++++++++++
+
+There are three ways to store non-frozen TensorFlow models.
+
+1. **SavedModel format**. In this case, a model consists of a special directory with a ``.pb`` file
+and several subfolders: ``variables``, ``assets``, and ``assets.extra``. For more information about the SavedModel directory, refer to the `README <https://github.com/tensorflow/tensorflow/tree/master/tensorflow/python/saved_model#components>`__ file in the TensorFlow repository.
+To convert such TensorFlow model, run the conversion similarly to other model formats and pass a path to the directory as a model argument:
+
+.. tab-set::
+
+   .. tab-item:: Python
+      :sync: py
+
+         import openvino as ov
+         ov_model = ov.convert_model('path_to_saved_model_dir')
+
+   .. tab-item:: CLI
+      :sync: cli
+
+         ovc path_to_saved_model_dir
+
+2. **Checkpoint**. In this case, a model consists of two files: ``inference_graph.pb`` (or ``inference_graph.pbtxt``) and ``checkpoint_file.ckpt``.
+If you do not have an inference graph file, refer to the `Freezing Custom Models in Python <#Freezing-Custom-Models-in-Python>`__  section.
+To convert the model with the inference graph in ``.pb`` format, provide paths to both files as an argument for `ovc` or `openvino.convert_model`:
+
+.. tab-set::
+
+   .. tab-item:: Python
+      :sync: py
+
+         import openvino as ov
+         ov_model = ov.convert_model(['path_to_inference_graph.pb', 'path_to_checkpoint_file.ckpt'])
+
+   .. tab-item:: CLI
+      :sync: cli
+
+         ovc path_to_inference_graph.pb path_to_checkpoint_file.ckpt
+
+To convert the model with the inference graph in ``.pbtxt`` format, specify the path to ``.pbtxt`` file instead of ``.pb`` file. The conversion API automatically detects the format of the provided file, there is no need to specify the model file format explicitly when calling `ovc` or `openvino.convert_model` in all examples in this document.
+
+3. **MetaGraph**. In this case, a model consists of three or four files stored in the same directory: ``model_name.meta``, ``model_name.index``,
+``model_name.data-00000-of-00001`` (the numbers may vary), and ``checkpoint`` (optional).
+To convert such a TensorFlow model, run the conversion providing a path to `.meta` file as an argument:
+
+.. tab-set::
+
+   .. tab-item:: Python
+      :sync: py
+
+         import openvino as ov
+         ov_model = ov.convert_model('path_to_meta_graph.meta')
+
+   .. tab-item:: CLI
+      :sync: cli
+
+         ovc path_to_meta_graph.meta
+
+
+Freezing Custom Models in Python
+++++++++++++++++++++++++++++++++
+
+When a network is defined in Python code, you must create an inference graph file. Graphs are usually built in a form
```

Comment:
Not quite sure about what case does it cover? Why do we need `Freezing Custom Models in Python`?

Reply:
I don't know. Looks like we a referring to this part from somewhere else. Check the document.